projects/libmaxminddb/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://www.maxmind.com"
+language: c
+primary_contact: "oschwald@gmail.com"
```

Comment:
@horgh I couldn't find your email ID, so I added @oschwald as primary. It would be great if I could add you as well.

Reply:
wstorey@maxmind.com. And for oschwald, could you use goschwald@maxmind.com instead please? Thank you!

Reply:
> [wstorey@maxmind.com](mailto:wstorey@maxmind.com). And for oschwald, could you use [goschwald@maxmind.com](mailto:goschwald@maxmind.com) instead please? Thank you!

Done